src/SFA.DAS.Forecasting.Api.UnitTests/Controllers/AccountProjection/WhenGettingProjectionSummaryForAnAccount.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Forecasting.Api.Controllers;
+using SFA.DAS.Forecasting.Api.Models;
+using SFA.DAS.Forecasting.Application.AccountProjection.Queries;
+
+namespace SFA.DAS.Forecasting.Api.UnitTests.Controllers.AccountProjection
+{
+    public class WhenGettingProjectionSummaryForAnAccount
+    {
+        private AccountProjectionController _accountProjectionController;
+        private Mock<IMediator> _mediator;
+        private GetAccountProjectionSummaryResult _accountSummaryResult;
+        private const long ExpectedAccountId = 123234;
+        private const decimal ExpectedFundsIn = 1110;
+        private const decimal ExpectedFundsOut = 9990;
+        private const int NumberOfMonths = 12;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _mediator = new Mock<IMediator>();
+            _accountSummaryResult = new GetAccountProjectionSummaryResult {
+                AccountId = ExpectedAccountId,
+                FundsIn = ExpectedFundsIn,
+                FundsOut = ExpectedFundsOut,
+                NumberOfMonths = 12,
+                ProjectionStartDate = DateTime.UtcNow
+            };
+
+            _mediator.Setup(x => x.Send(It.Is<GetAccountProjectionSummaryQuery>(c => c.AccountId.Equals(ExpectedAccountId)), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(_accountSummaryResult);
+
+            _accountProjectionController = new AccountProjectionController(_mediator.Object);
+        }
+
+        [Test]
+        public async Task Then_The_Projections_Are_Returned()
+        {
+            //Act
+            var actual = await _accountProjectionController.GetProjectedFundingSummary(ExpectedAccountId, NumberOfMonths);
+
+            //Assert
+            Assert.IsNotNull(actual);
+            var result = actual as ObjectResult;
+            Assert.IsNotNull(result?.StatusCode);
+            Assert.AreEqual(HttpStatusCode.OK, (HttpStatusCode)result.StatusCode);
+            Assert.IsNotNull(result.Value);
+            var actualSummaryResult = result.Value as GetAccountProjectionSummaryResult;
+            Assert.IsNotNull(actualSummaryResult);
```

Comment:
could we use FluentAssertions to check weather the actualSummaryResult is equivalent to _accountSummary result. That would cover the bottom 3 asserts

Reply:
I've written the asserts to match the existing tests which don't use FluentAssertions .